### PR TITLE
/start & /help command

### PR DESCRIPTION
**Pull request by alexshavelev44**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ Python 3.7
 	-   SOUNDCLOUD_CLIENT_ID - API key for SoundCloud API
 
 -   Optional environment variables:
-    	-   BOT_ADMINS_CHAT - if you set admins chat (eg @our_chat) bot messages will have 'report button'. if click - bot forwards messages to admins chat
+    	
+	-   BOT_ADMINS_CHAT - if you set admins chat (eg @our_chat) bot messages will have 'report button'. if click - bot forwards messages to admins chat
+	-    EXAMPLE_MEDIA - if you example photo/gif URL it will be shown on /start and /help user commands
```
Comment (Ignisor) [inline README.md]: `-    EXAMPLE_MEDIA` - looks like too many spaces

File: interfaces/telegram.py [hunk 2]
```diff
@@ -23,6 +29,11 @@ def _init_handlers(self):
                 self._handle_message
             ),
 
+            MessageHandler(
```
Comment (Ignisor) [inline interfaces/telegram.py]: https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.commandhandler.html - there is a CommandHandler for commands

File: README.md (hunk 1 above)
Comment (alexshavelev44) [reply README.md]: it's ok 
image

File: interfaces/telegram.py (hunk 2 above)
Comment (alexshavelev44) [reply interfaces/telegram.py]: fxd

**Pull request merged**